mnefun/_mnefun.py

```diff
@@ -262,6 +262,12 @@ class Params(Frozen):
         Default is ('mag', 'grad', 'eeg'). Can set to ('mag', 'grad', 'eeg',
         'eog) to use EOG channel rejection criterion from autoreject module to
         reject trials on basis of EOG.
+    mf_autobad : bool
+        Default False. If True use Maxfilter to automatically mark bad
+        channels prior to SSS denoising.
+    mf_badlimit : int
+        Default is 7. Threshold limit for Maxfilter noisy channel
+        detection.
```

Comment:
What is the default value in `maxfilter`? We should use whatever their default is, probably.

Reply:
7 is Default. I used five the test.

Reply:
Actually, the default value 7 seems to work reasonably well for infant data, maybe just get rid of it?

Reply:
Sure